lib/sikuli/outlook.sikuli/outlook.py

```diff
@@ -45,9 +45,10 @@ class outlook(WebApp):
     ]
 
     def wait_for_loaded(self, similarity=0.70):
-        self._wait_for_loaded(component=outlook.OUTLOOK_ACCOUNT_AVATAR, similarity=similarity, timeout=60)
-        self._wait_for_loaded(component=outlook.OUTLOOK_MIDDLE_UPPER_MENU_ICON, similarity=similarity, timeout=60)
-        return self._wait_for_loaded(component=outlook.OUTLOOK_LEFT_UPPER_MENU_ICON, similarity=similarity, timeout=60)
+        return self._wait_for_loaded(component=outlook.OUTLOOK_COMPOSE_NEW_MAIL_ICON, similarity=similarity, timeout=60)
```

Comment:
You should not remove line 48-50 due to compose new mail icon and the other icon didn't show by certain sequence.